Resources/Prototypes/Loadouts/Jobs/Medical/chief_medical_officer.yml

```diff
@@ -59,6 +69,19 @@
   equipment:
     neck: ClothingNeckMantleCMO
 
+- type: loadout
+  id: CMOGreatCloak
+  equipment:
+    neck: ClothingNeckCloakCmosgreatcloak
+  effects:
+  - !type:GroupLoadoutEffect
+    proto: PrestigeCMO
+
+- type: startingGear
```

Comment:
this is just for visitor roles and other entities that get spawned with clothes, so this isn't required